README.md

```diff
@@ -20,3 +20,60 @@ The initial password can be changed in [`docker-compose.yml`](docker-compose.yml
 For more configuration options see:
 - https://github.com/ome/omero-server-docker/blob/master/README.md
 - https://github.com/ome/omero-web-docker/blob/master/README.md
+
+## User management
+
+In the `ldap` container, you can use the pre-installed `ldapmanager` tool
+from the [apacheds-docker](https://github.com/ome/apacheds-docker/) repository
+to set up users and groups:
+
+```
+docker-compose exec ldap bash
+ldapmanager init
+ldapmanager user u1 --password u1
+ldapmanager group u1 --member u1
+```
+
+Additionally, in the `ldap` container, you can modify a user using `ldapmodify`:
+
+```
+ldapmodify -H ldap://localhost:10389 -D uid=admin,ou=system -W < update.ldif
+```
+with this being an example of `update.ldif` file:
+
+```
+version: 1
+dn: uid=t,ou=Users,dc=openmicroscopy,dc=org
+changetype: modify
+replace: givenName
+givenName: X.
+```
+
+## Testing with omero-ldaptool
+
+[omero-ldap](https://github.com/glencoesoftware/omero-ldaptool) provides a way to check if your
+OMERO configuration is talking to LDAP correctly.
+
+Then from the `omeroserver` container, download the configuration:
+
+```
+docker-compose exec omeroserver /opt/omero/server/venv3/bin/omero config get --show-password > cfg
+```
+
+Then, run the build and the tool:
+
+```
+./gradlew installDist
```

Comment:
This raises the question of whether the tool could be pre-built into the server image.

Reply:
or be its own image?